packages/bspwm/common.patch

```diff
@@ -0,0 +1,11 @@
+--- src/common.h	2020-08-02 15:51:32.000000000 +0800
++++ src.mod/common.h	2022-03-21 21:24:04.760854462 +0800
+@@ -25,7 +25,7 @@
+ #ifndef BSPWM_COMMON_H
+ #define BSPWM_COMMON_H
+ 
+-#define SOCKET_PATH_TPL  "/tmp/bspwm%s_%i_%i-socket"
++#define SOCKET_PATH_TPL  "/data/data/com.termux/files/usr/tmp/bspwm%s_%i_%i-socket"
```

Comment:
Please replace `/data/data/com.termux/files/usr` with `@TERMUX_PREFIX@`, it is automatically sed'ed when the [patch is applied](https://github.com/termux/termux-packages/blob/master/scripts/build/termux_step_patch_package.sh#L27)

Reply:
Helpers.c also same?

Reply:
Yes